client/client.go

```diff
@@ -0,0 +1,164 @@
+package client
```

Comment:
Probably a nitpick from me, but I would prefer to have a `daemon` folder that has the RPC API and the Respective client rather than having two folders at the root.

This would follow the IPFS approach where there is a package for the "Core" of the service/implementation, and then daemon & clients are ways to interact with that Core.

Reply:
You're right `client` should not have been top level - moved it to `pkg/daemon/client` as suggested.